### PR TITLE
Remove legacy deposit/withdrawal in Electra

### PR DESCRIPTION
**Pull request by qqqeck**
Remove legacy deposit/withdrawal from Electra

- [ ] remove legacy fields(eth1data, deposit, bls_to_execution) from electra block
- [ ] remove unused fiels from electra state
- [ ] disable powchain
- [ ] remove index field from deposit_request ( should be handled together w/ geth )
- [ ] fix tests

Temporarily commented out tests
- TestProcessDeposit_SkipsInvalidDeposit
- TestVerifyOperationLengths_Electra
- TestProposer_PendingDeposits_Electra

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Enable Powerlevel10k instant prompt. Should stay close to the top of ~/.zshrc.
```
Comment (gazzua) [inline 1:1]: i think you committed wrong file 😄

File: proto/prysm/v1alpha1/sync_committee.pb.go [hunk 2]
```diff
@@ -0,0 +1,487 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```
Comment (gazzua) [inline proto/prysm/v1alpha1/sync_committee.pb.go:1]: committed wrong file 😄

File: proto/eth/v2/sync_committee.pb.go [hunk 3]
```diff
@@ -0,0 +1,340 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```
Comment (gazzua) [inline proto/eth/v2/sync_committee.pb.go:1]: committed wrong file. sync committee is deleted in other pr

**Comment by gazzua** [comment]
add state version check would be better at `state-native/setters-eth1.go` and `state-native/getters-eth1.go`
```
if b.version >= version.Electra {
   return errNotSupported("Eth1Data", b.version)
}
```

File: beacon-chain/state/state-native/beacon_state.go [hunk 4]
```diff
@@ -57,7 +57,6 @@ type BeaconState struct {
 	nextWithdrawalValidatorIndex        primitives.ValidatorIndex
 
 	// Electra fields
-	depositRequestsStartIndex uint64
 	depositBalanceToConsume   primitives.Gwei
```
Comment (gazzua) [inline beacon-chain/state/state-native/beacon_state.go:60]: `DepositRequestsStartIndex` in `beaconStateMarshalable` should also be removed.

**Comment by qqqeck** [comment]
Closed due to a change of plan

**Pull request closed without merging**